### PR TITLE
Add versioned _ping endpoint

### LINKED ISSUE #7008: Incompatibility with the "docker-py" library because /_ping is not versioned (related to #6548)
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**
(This has been previously reported at #6548 but that issue is closed)

I was trying to use the [aws-sam-cli](https://github.com/awslabs/aws-sam-cli) tool when I got complained that it couldn't talk with docker.
It uses [docker-py](https://github.com/docker/docker-py) internally and ultimately calls this line
https://github.com/docker/docker-py/blob/master/docker/api/daemon.py#L166
to check whether docker is reachable or not.

However, `self._url('/_ping')` returns `'http+docker://localhost/v1.35/_ping'` which is currently not supported by podman.

You may find the following snippet useful:
```
$ podman system service --time=0 &
$ export DOCKER_HOST=unix:///run/user/1000/podman/podman.sock
$ python
Python 3.8.3 (default, May 29 2020, 00:00:00) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import docker
>>> client = docker.from_env()
>>> client.api._url('/_ping')
'http+docker://localhost/v1.35/_ping'
>>> client.ping()
Traceback (most recent call last):
  File "/home/arnold/.local/lib/python3.8/site-packages/docker/api/client.py", line 261, in _raise_for_status
    response.raise_for_status()
  File "/home/arnold/.local/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http+docker://localhost/v1.35/_ping

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/arnold/.local/lib/python3.8/site-packages/docker/client.py", line 187, in ping
    return self.api.ping(*args, **kwargs)
  File "/home/arnold/.local/lib/python3.8/site-packages/docker/api/daemon.py", line 166, in ping
    return self._result(self._get(self._url('/_ping'))) == 'OK'
  File "/home/arnold/.local/lib/python3.8/site-packages/docker/api/client.py", line 267, in _result
    self._raise_for_status(response)
  File "/home/arnold/.local/lib/python3.8/site-packages/docker/api/client.py", line 263, in _raise_for_status
    raise create_api_error_from_http_exception(e)
  File "/home/arnold/.local/lib/python3.8/site-packages/docker/errors.py", line 31, in create_api_error_from_http_exception
    raise cls(e, response=response, explanation=explanation)
docker.errors.NotFound: 404 Client Error: Not Found ("b'Not Found'") 
```

**Steps to reproduce the issue:**

Run
```
podman system service --time=0 &
curl --unix-socket /run/user/1000/podman/podman.sock http://d/v1.35/_ping
```

**Describe the results you received:**
`Not Found`

**Describe the results you expected:**
`OK`, as is returned by `curl --unix-socket /run/user/1000/podman/podman.sock http://d/_ping`

~**Additional information you deem important (e.g. issue happens only occasionally):**~

**Output of `podman version`:**

```
Version:      2.0.2
API Version:  1
Go Version:   go1.14.3
Built:        Thu Jan  1 09:00:00 1970
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

```
host:
  arch: amd64
  buildahVersion: 1.15.0
  cgroupVersion: v2
  conmon:
    package: conmon-2.0.18-1.fc32.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.0.18, commit: 6e8799f576f11f902cd8a8d8b45b2b2caf636a85'
  cpus: 8
  distribution:
    distribution: fedora
    version: "32"
  eventLogger: file
  hostname: xps-9300
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  kernel: 5.7.7-200.fc32.x86_64
  linkmode: dynamic
  memFree: 1243295744
  memTotal: 16343982080
  ociRuntime:
    name: crun
    package: crun-0.14.1-1.fc32.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 0.14.1
      commit: 598ea5e192ca12d4f6378217d3ab1415efeddefa
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  os: linux
  remoteSocket:
    exists: true
    path: /run/user/1000/podman/podman.sock
  rootless: true
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns-1.1.4-1.fc32.x86_64
    version: |-
      slirp4netns version 1.1.4
      commit: b66ffa8e262507e37fca689822d23430f3357fe8
      libslirp: 4.3.1
      SLIRP_CONFIG_VERSION_MAX: 2
  swapFree: 8237608960
  swapTotal: 8241803264
  uptime: 207h 8m 14.25s (Approximately 8.62 days)
registries:
  search:
  - registry.fedoraproject.org
  - registry.access.redhat.com
  - registry.centos.org
  - docker.io
store:
  configFile: /home/arnold/.config/containers/storage.conf
  containerStore:
    number: 8
    paused: 0
    running: 1
    stopped: 7
  graphDriverName: overlay
  graphOptions:
    overlay.mount_program:
      Executable: /usr/bin/fuse-overlayfs
      Package: fuse-overlayfs-1.1.2-1.fc32.x86_64
      Version: |-
        fusermount3 version: 3.9.1
        fuse-overlayfs: version 1.1.0
        FUSE library version 3.9.1
        using FUSE kernel interface version 7.31
  graphRoot: /home/arnold/.local/share/containers/storage
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 40
  runRoot: /run/user/1000/containers
  volumePath: /home/arnold/.local/share/containers/storage/volumes
version:
  APIVersion: 1
  Built: 0
  BuiltTime: Thu Jan  1 09:00:00 1970
  GitCommit: ""
  GoVersion: go1.14.3
  OsArch: linux/amd64
  Version: 2.0.2
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-2.0.2-1.fc32.x86_64
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Physical

### PR DESCRIPTION
**Pull request by jwhonce**
Add versioned _ping endpoints

Fixes #7008

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by edsantiago** [comment]
Rebase to fix broken CI

**Comment by mheon** [comment]
LGTM once CI goes green

File: test/apiv2/rest_api/test_rest_v1_0_0.py [hunk 1]
```diff
@@ -205,7 +207,21 @@ def do_search():
         search.join(timeout=10)
         self.assertFalse(search.is_alive(), "/images/search took too long")
 
-    def validateObjectFields(self, buffer):
+    def test_ping(self):
```
Comment (vrothberg) [inline test/apiv2/rest_api/test_rest_v1_0_0.py:210]: Can we add a test to `test/apiv2` as well? Currently, `rest_api` is not firing in the CI and I want to be extra sure we don't regress in between merging and enabling the tests.

**Pull request merged**